ibtk/src/solvers/impls/PETScKrylovLinearSolver.cpp

```diff
@@ -209,7 +209,9 @@ PETScKrylovLinearSolver::solveSystem(SAMRAIVectorReal<NDIM, double>& x, SAMRAIVe
     d_A->imposeSolBcs(x);
 
     // Get iterations count and residual norm.
-    ierr = KSPGetIterationNumber(d_petsc_ksp, &d_current_iterations);
+    PetscInt current_iter = d_current_iterations;
```

Comment:
Should we go ahead and systematically use PETSc `typedef`s?

Reply:
That would be nice to do but it doesn't work well with SAMRAI - there are places where we need to store index information but SAMRAI doesn't support `long int`.

Reply:
Should we drop this change since we are not changing the type of `d_current_iterations`? If `PetscInt` is different from `int`, we are already going to run into a lot of problem. 😬

Reply:
Yeah, I guess we can just drop this commit - its only a partial fix for a much more substantial problem.